cmd/virt-handler/Dockerfile

```diff
@@ -23,9 +23,20 @@ FROM centos:7
 # no-lower than what was used when the binary was built.
 RUN yum -y install wget && \
     wget -P /etc/yum.repos.d https://copr.fedorainfracloud.org/coprs/jmliger/virt7-upstream/repo/epel-7/jmliger-virt7-upstream-epel-7.repo  \
-    && yum -y install libvirt-client \
+    && yum -y install libvirt-client cyrus-sasl-md5 \
     && yum -y clean all
 
+COPY auth.conf /etc/libvirt/auth.conf
 COPY virt-handler /virt-handler
+COPY virt-handler.sh /virt-handler.sh
+RUN chmod a+x /virt-handler.sh
 
-ENTRYPOINT [ "/virt-handler" ]
+RUN useradd -u 1000 --create-home -s /bin/bash virt-handler && \
+    chown virt-handler /etc/libvirt && \
+    chown virt-handler /etc/libvirt/auth.conf && \
+    chown virt-handler /virt-handler.sh
```

Comment:
Ewww, should not be changing any of these file owners. Leave them all as their normal user. The auth.conf file should be in $HOME/.config/libvirt/auth.conf of the virt-handler user account, not /etc/libvirt